stdlib/REPL/src/REPL.jl

```diff
@@ -765,15 +765,21 @@ setup_interface(
     repl::LineEditREPL;
     # those keyword arguments may be deprecated eventually in favor of the Options mechanism
     hascolor::Bool = repl.options.hascolor,
-    extra_repl_keymap::Union{Dict,Vector{<:Dict}} = repl.options.extra_keymap
+    extra_repl_keymap::Any = repl.options.extra_keymap
```

Comment:
`@nospecialize`? Just asserting that it's an `Any` doesn't prevent the compiler from trying to specialize it. Same for the one below.

Reply:
I think this takes care of that:

https://github.com/JuliaLang/julia/blob/5e75cfe24ac65400b8faa2972c0ac8f8b1faa9dd/stdlib/REPL/src/REPL.jl#L50